src/core/streaming_playback_manager.py

```
@@ -1274,6 +1274,20 @@ def _ensure_startup_ready(
+            self._startup_ready[call_id] = True
+            stream_info['startup_ready'] = True
+            try:
+                logger.info(
+                    "Streaming startup released short final segment",
+                    call_id=call_id,
+                    stream_id=stream_id,
+                    buffered_frames=available_frames,
+                    min_start_chunks=min_need,
+                )
+            except Exception:
+                pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)